src/Components/About.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import userinfo from './users';
+
+export default class About extends React.Component {
+     
+    constructor(props) {
+        super(props);
+        this.state = {
+            userID: 0,
+        }
+    }
+
+    render() {
+        return (
+            <div className="about">
+                <div className="basic-info">
+                    <div className="header">
+                        BASIC INFORMATION
+                    </div>
+                    <div className="content">
+                        <div className="row">
+                            <div className="title col-xs-4 col-sm-4 col-md-4 col-lg-4">
+                                Full Name 
+                            </div>
+                            <div className="value col-xs-8 col-sm-8 col-md-8 col-lg-8">
```

Comment:
if you used col-xs-8 do we still need  col-sm-8 col-md-8 col-lg-8 ??

Reply:
No, mam. I forgot and kept following the same throughout the project. I am solving this issue.